### PR TITLE
KFSPTS-17852 upgrade to the 10/31/2019 version of financials

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml [hunk 1]
```diff
@@ -46,20 +31,6 @@
 			class="edu.cornell.kfs.sys.document.validation.impl.AccountingLineRequiredOverridesValidation" abstract="true">
 		<property name="accountingLineRuleHelperService" ref="purchaseOrderAmendmentAccountingLineRuleHelperService"/>
 	</bean>
-
-	<bean id="Requisition-newIndividualItemValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.RequisitionNewIndividualItemValidation" abstract="true">
```
Comment (cah292) [inline src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml:50]: The custom cu-kfs version of RequisitionNewIndividualItemValidation does not extend from the one in base code, and it does not have the commodityCodeIsRequired() method override like in the base code's class. Can you please confirm if we should be using the base code's variant instead? If so, then can we remove the cu-kfs version of RequisitionNewIndividualItemValidation?

Comment (db483) [reply src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml:50]: Restoring entry for RequisitionNewIndividualItemValidation

File: src/main/java/org/kuali/kfs/module/purap/service/impl/ElectronicInvoiceHelperServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,1886 @@
...
+    protected DateTimeService dateTimeService;
+    protected ParameterService parameterService;
+    protected DocumentService documentService;
+    protected AttachmentService attachmentService;
+    protected NoteService noteService;
+    protected BankService bankService;
+    protected KualiRuleService kualiRuleService;
+    private BusinessObjectService businessObjectService;
+    private DataDictionaryService dataDictionaryService;
+    //CU customization change from priate to protected
```
Comment (cah292) [inline src/main/java/org/kuali/kfs/module/purap/service/impl/ElectronicInvoiceHelperServiceImpl.java:157]: Minor typo in comment.

Comment (db483) [reply src/main/java/org/kuali/kfs/module/purap/service/impl/ElectronicInvoiceHelperServiceImpl.java:157]: updated

**Comment by db483** [comment]
I've added the requested changes, please review again. Thank you.

**Pull request merged**